client/src/containers/edit/EditEducation.js

```diff
@@ -0,0 +1,114 @@
+import React, {Component} from 'react'
+import {
+ MenuItem,
+ TextField,
+ SelectField,
+ DatePicker,
+ RaisedButton
+} from 'material-ui'
+import Save from 'material-ui/svg-icons/content/save'
+import ContentAdd from 'material-ui/svg-icons/content/add'
```

Comment:
these two lines could be condensed into
`import { Save, ContentAdd } from 'material-ui/svg-icons/content'`